src/doc/trpl/if.md

```diff
@@ -34,6 +34,20 @@ if x == 5 {
 }
 ```
 
+If there is more than one case, use an `else if`:
+
+```rust
+let x = 5;
+
+if x == 5 {
+    println!("x is five!");
+} else if x == 6 {
+    println!("x is six!");
+} else {
+    println!("x is not five or six :(");
```

Comment:
FYI https://english.stackexchange.com/questions/10049/usage-of-neither-nor-versus-not-or

I'm not going to spend too much time defending my suggestion here, but I stand by `not...nor`